mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,404 @@
+    cnt_info.update(baselines=np.array(baselines), n_samples=n_samples,
+                    stim_channel=stim_channel)
+    info.update(filename=input_fname, meas_date=np.array([meas_date, 0]),
+                description=str(session_label), buffer_size_sec=10., bads=bads,
+                subject_info=subject_info, chs=chs)
+    info._check_consistency()
+    return info, cnt_info
+
+
+def _topo_to_sphere(pos, eegs):
+    """Helper function for transforming xy-coordinates to sphere.
+    Parameters
+    ----------
+    pos : array of shape (chs, 2)
+        xy-oordinates to transform.
+    eegs : list of int
+        Indices of eeg channels that are included when calculating the sphere.
+
+    Returns
+    -------
+    coords : list of shape (chs, 3)
+        xyz-coordinates.
+    """
+    xs = np.array(pos)[:, 0]
+    ys = np.array(pos)[:, 1]
+
+    sqs = max(np.sqrt((xs[eegs] ** 2) + (ys[eegs] ** 2)))
+    xs /= sqs  # Shape to a sphere and normalize
+    ys /= sqs
+
+    xs += 0.5 - np.mean(xs[eegs])  # Centralize the points
+    ys += 0.5 - np.mean(ys[eegs])
+
+    xs = xs * 2 - 1  # Values ranging from -1 to 1
+    ys = ys * 2 - 1
+
+    coords = list()
+    for x, y in zip(xs, ys):
+        r = np.sqrt(x ** 2 + y ** 2)
+        if r > 1:  # If a point is outside the sphere set z=0.
+            coords.append([x, y, 0])
+            continue
+        alpha = np.arccos(r)
+        z = np.sin(alpha)
+        coords.append([x, y, z])
+    return coords
+
+
+class RawCNT(_BaseRaw):
+    """Raw object from Neuroscan CNT file.
+
+    Note: Channels that are not assigned with keywords ``eog``, ``ecg``,
+    ``emg`` and ``misc`` are assigned as eeg channels. All the eeg channels are
+    fit to a sphere when computing the z-coordinates for the channels.
+    If channels assigned as eeg channels were placed away from the head (i.e.
+    x and y coordinates don't fit to a sphere), all the channel locations will
+    be distorted.
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the CNT file.
+    eog : list | tuple
+        Names of channels or list of indices that should be designated
+        EOG channels. If 'auto', the channel names beginning with
+        ``EOG`` are used. Defaults to empty tuple.
+    ecg : list or tuple
+        Names of channels or list of indices that should be designated
+        ECG channels. If 'auto', the channel names beginning with
+        ``ECG`` are used. Defaults to empty tuple.
+    emg : list or tuple
+        Names of channels or list of indices that should be designated
+        EMG channels. If 'auto', the channel names beginning with
+        ``EMG`` are used. Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. Defaults to empty tuple.
+    read_blocks : bool
+        Whether to read data in blocks. This is for dealing with different
+        kinds of CNT data formats.
```

Comment:
can you clarify? should the user have to know how to set this value? can it be inferred from the data?